pcsx2/gui/AppConfig.cpp

```diff
@@ -758,6 +761,18 @@ void AppConfig::FolderOptions::LoadSave( IniInterface& ini )
 	 //  --> on load, these relative paths will be expanded relative to the exe folder.
 	bool rel = ( ini.IsLoading() || IsPortable() );
 	
+	//Bios = Path::isPortable(Bios, IsPortable());
+	Snapshots = Path::isPortable(Snapshots, IsPortable());
+	Savestates = Path::isPortable(Savestates, IsPortable());
+	MemoryCards = Path::isPortable(MemoryCards, IsPortable());
+	Logs = Path::isPortable(Logs, IsPortable());
+	Langs = Path::isPortable(Langs, IsPortable());
+	Cheats = Path::isPortable(Cheats, IsPortable());
+	CheatsWS = Path::isPortable(CheatsWS, IsPortable());
+	/*RunIso = Path::isPortable(RunIso, IsPortable());
+	RunELF = Path::isPortable(RunELF, IsPortable());
+	RunDisc = Path::isPortable(RunDisc, IsPortable());*/
```

Comment:
Marking these lines so i know to verify that this functionality still works, if so these lines can be removed.

Reply:
These lines should be able to removed, i don't have a Disc drive so i can't verify the third one, but RunISO and RunELF get updated appropriately:
![image](https://user-images.githubusercontent.com/13153231/105564625-0246cc00-5cf1-11eb-94ce-7be1b4a36157.png)